worlds/rabi_ribi/client/memory_io.py

```diff
@@ -346,3 +351,15 @@ def is_in_costume_menu(self) -> bool:
         True if the player is currently selecting a costume
         """
         return self._read_4_byte_bool(OFFSET_IN_COSTUME_MENU)
+
+    def has_zero_health(self) -> bool:
+        """
+        True if the player's health is at 0
+        """
+        return not self._read_4_byte_bool(OFFSET_MAX_HEALTH)
+
+    def set_player_health_to_zero(self):
+        """
+        Sets the player health to 0
+        """
+        self.rr_mem.write_bytes(self._read_int(OFFSET_CURRENT_HEALTH_A) + OFFSET_CURRENT_HEALTH_B, b'\x00\x00\x00\x00', 4)
```

Comment:
There should be a `write_int()` method available on the `Pymem` object.
(nit) We should separate this into two lines.